ddtrace/contrib/falcon/middleware.py

```diff
@@ -19,6 +22,13 @@ def __init__(self, tracer, service="falcon"):
         )
 
     def process_request(self, req, resp):
+        if self._distributed_tracing:
+            # Falcon uppercases all header names.
+            headers = dict((k.lower(), v) for k, v in req.headers.items())
+            propagator = HTTPPropagator()
```

Comment:
Should I make propagator a member variable? Its methods may as well be static (no state), but aren't for some reason. Just in case state is added in the future I thought it would be safe to make a new one each time.

Reply:
Good question. Honestly I think they should be static methods since you don't preserve the state between executions, but probably there is something we may need to change in the future for OpenTracing compatibility.

I would say to create a new one until we make a decision, just to keep a consistent approach among integrations. Then we'll update all of them at once.